network-api/networkapi/buyersguide/templates/buyersguide_home.html

```diff
@@ -87,6 +87,12 @@ <h1 class="h1-heading text-center mt-4">Shop Safe This Holiday Season</h1>
         </a>
       </figure>
       {% endfor %}
+      <div class="no-matching-products-note bg-gray w-100 pt-5 d-none">
```

Comment:
Let's make this part of the filter component. It can already know if there are no results by checking how many products are visible after the visibility pass.